xrmocap/core/evaluation/__init__.py

```diff
@@ -0,0 +1,9 @@
+# yapf: disable
+from xrmocap.core.evaluation.mvp_evaluation import MVPEvaluation
+from xrmocap.core.evaluation.top_down_association_evaluation import (
+    TopDownAssociationEvaluation,
+)
+
+# yapf: enable
+
+__all__ = ['MVPEvaluation', 'TopDownAssociationEvaluation']
```

Comment:
Is this essential for readthedocs? `__init__.py` import in estimation is OK because that one is deepest layer. We may have some packages for evaluation inside `xrmocap.core.evaluation`, and import of those common functions will go through these lines.

Reply:
As offline discussion, it is essential for readthedocs to find what we want to show. It depends on our design to locate a propriate place for `__init__.py`.